examples/time_series/timeseries_example.py

```diff
@@ -1,7 +1,7 @@
 """
-====================================
-Using `~sunpy.timeseries.TimeSeries`
-====================================
+===================
+Using `.TimeSeries`
```

Comment:
Still looks a bit odd on the gallery page, get rid of the link in the title altogether?
<img width="182" alt="Screenshot 2022-11-06 at 12 18 50" src="https://user-images.githubusercontent.com/6197628/200170340-0407df07-e7b3-44b5-a7ed-85a255ae6a49.png">

Reply:
Yeah, it does look pretty bad.